ssl.pl

```diff
@@ -312,7 +312,30 @@
 
 ssl_context(Role, SSL, Module:Options) :-
     select_option(ssl_method(Method), Options, O1, sslv23),
-    '_ssl_context'(Role, SSL, Module:O1, Method).
+    upgrade_legacy_options(O1, O2),
+    % If the options list does not contain a cacerts/1 term (including cacerts([]))
+    % then assume that the system root certificates should be used
+    (  memberchk(cacerts(_), O2)
```

Comment:
This should be option/2, no?

Reply:
If you prefer option(cacerts(_), O2, _) to memberchk(cacerts(_), O2) then that's fine by me. The point is that we just want to check if an cacerts/1 term is in there - and only do something if it is *not*

Reply:
I know.  option/2 is more explicit we are processing options and allows for the alternative dict based option format and/or other future changes.